views/js/pciCreator/dev/likertScaleInteraction/runtime/css/likertScaleInteraction.css

```diff
@@ -5,22 +5,68 @@
     list-style-type: none;
     margin: 0 5px;
     padding: 0;
-    display:inline-block;
+    display: inline-block;
     position: relative;
-    top:7px;
+    top: 7px;
+    flex-shrink: 0;
 }
 
 .qti-customInteraction .likertScaleInteraction ul.likert li{
     float: left;
     text-align: left;
     list-style-type: none;
 }
+.grid-row[dir=rtl] .qti-customInteraction .likertScaleInteraction ul.likert li{
```

Comment:
Yes, indeed, I would rather not use `.grid-row`, no matter this is a PCI or another interaction. This is not a standard class and we should avoid rely on it.

Moreover, the direction attribute is not set at that level, but on the item body

Reply:
If you want to support bidirectional content, you'll have to wrap any CSS directive in dual mode ltr & default / rtl.
In Sass this can be achieved using simple mixins, for example : 
https://gist.github.com/krampstudio/ae7f995cee5e68bd7ab84c8df7b12390